### ISSUE TITLE
Hyperdynamic left ventricular ejection fraction

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7588]
**Preferred term label:**
Hyperdynamic left ventricular ejection fraction

**Synonyms**
High ejection fraction

**Definition (free text, please give PubMed ID)**
An abnoramlly increased ejection fraction of the left ventricle, usually defined as a left ventricular ejection fraction greater than 70 percent.
PMID:26250903
PMID:29083812

See also https://pubmed.ncbi.nlm.nih.gov/29083812/ to improve defintions

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
OMIM:613255 
PMID: 16236538  1/1

**Comment by pnrobinson** [comment]
added
